tests/bdd/features/snapshot/delete/delete.feature

```diff
@@ -0,0 +1,73 @@
+Feature: Volume Snapshot deletion
+
+  Background:
+    Given a deployer cluster
+
+  # USUAL DELETE OPERATION
+  Scenario Outline: Snapshot deletion volume
+    Given a <publish_status> volume with single replica
+    And a snapshot is created for the volume
+    When the created snapshot is deleted
+    Then the snapshot deletion should not fail
+    And the snapshot should not be present upon listing
+    Examples:
+      | publish_status |
+      | published      |
+      | unpublished    |
+
+
+  # USUAL DELETE OPERATION WITHOUT SOURCE
+  Scenario Outline: Snapshot deletion volume after volume deletion
+    Given a <publish_status> volume with single replica
+    And a snapshot is created for the volume
+    And the source volume is deleted
+    When the created snapshot is deleted
+    Then the snapshot deletion should not fail
+    And the snapshot should not be present upon listing
+    Examples:
+      | publish_status |
+      | published      |
+      | unpublished    |
+
+
+  # SOURCE DELETION WHILE IN PROGRESS (could be flaky ?)
```

Comment:
Depends how we implement "creation is on progress"

Reply:
I was thinking of launching two threads but not sure how that would end up until we implement.

Reply:
Removing this for now.